packages/fxa-settings/src/components/FormVerifyTotp/index.test.tsx

```diff
@@ -22,15 +22,15 @@ describe('FormVerifyTotp component', () => {
       renderWithLocalizationProvider(<Subject />);
       const button = screen.getByRole('button');
       expect(button).toHaveTextContent('Submit');
-      expect(button).toBeDisabled();
+      expect(button).toHaveAttribute('aria-disabled', 'true');
```

Comment:
Is this only needed because this had an `aria-label` on it, so it reads funky without `aria-disabled`? I would have assumed that just the `disabled` reading "dimmed" would be OK, but LMK if it seems like we need to always add this otherwise since we should have another ticket for that. Disabled buttons read OK in VO but I haven't tested in Narrator.

Reply:
Disabled buttons are taken out of focus order though. Are you able to reach it easily when it's disabled (not aria-disabled)? 

VO reading "disabled" as "dimmed" appears to be fine and expected, not concerned about that part.

And good point, if this solution is approved here to improve accessibility for disabled buttons, I'll file a ticket to review our other disabled buttons.

Reply:
When I was testing the connect another device / pairing flow, yes, I could reach a disabled button with VO on a Mac reading "dimmed" without `aria-disabled` being `true`/`false`, but I didn't test with any other assistive tech, and I didn't tab to it, I was reading element by element. I definitely agree it needs to be able to be reached or users might miss that it ever exists. 

[I'm reading this](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled):
> When needing to disable native HTML form controls, developers will need to specify the disabled attribute, as it provides all of the generally expected features of disabling a control by default. However, there can be instances where elements need to be exposed as disabled, but are still available for users to find when navigating via the Tab key. Doing so can improve their discoverability as they will not be removed from the focus order of the web page, as aria-disabled does not change the focusability of such elements, nor will the elements be dimmed by default browser styling, making them easier to read. 

So, maybe the right thing to do to cover all screen readers is to use `aria-disabled` instead of `disabled` so it's _tabbable_, and update our classes as well, since we use `:disabled` in our CSS.